apps/web/src/legacy/pages/HomePage.tsx

```diff
@@ -1,7 +1,5 @@
-import { Button } from '@mantine/core';
-
 function HomePage() {
-  return <Button>s</Button>;
+  return null;
```

Comment:
Just one question if you don't mind is there was any special reason for the change in the legacy code?

Reply:
Just reverted a change I did for a test before.